packages/extension/webpack.config.js

```diff
@@ -89,11 +89,12 @@ module.exports = {
                     const jsonString = JSON.stringify(
                         {
                             ...manifest,
-                            version:
-                                process.env.PL_BUILD_ENV === "Production"
-                                    ? build
-                                    : `${vendorVersion}.${devBuildVersion}`,
-                            version_name: vendorVersion,
+                            version: `${process.env.PL_VENDOR_VERSION || version}.${
+                                ["Production", "Beta"].includes(process.env.PL_BUILD_ENV)
+                                    ? process.env.RELEASE_BUILD
+                                    : devBuildVersion
```

Comment:
It would really be a lot nicer if we could have the build number increment with each build. Is there a way to store the last build number somewhere and update it on every build without introducing a lot of noise to the commit history? Maybe with [git notes](https://git-scm.com/docs/git-notes)? Or a tag? Or within a text file "attached" to the last build as an artifact? Some place where it can easily be retrieved, incremented and updated from the github action.

Reply:
It's not easily possible and I think that's _way_ overkill. I'd be fine with just deleting the `devBuildVersion` logic since we no longer run these automatically.

A good way would be to have Firefox tell us which was the last version, but I can't find anyway to get that information for unlisted builds.

Reply:
Fair enough. We certainly shouldn't over-engineer it just because of Firefox' stupid rules.